app/views/events/show.html.erb

```diff
@@ -22,6 +22,27 @@
   <% end %>
 </div>
 
+<% if @event.is_a? Tournament and @event.finale.present? and @event.finale.has_winner? %>
+  <div class="table-responsive">
+    <h3><%= t('events.show.winners') %></h3>
+    <table class="table">
+      <tr>
+        <th>Platzierung</th>
+        <th><%= @event.single? ? t('.title', default: User.model_name.human.titleize) : t('.title', default: Team.model_name.human.titleize)%></th>
```

Comment:
Is there no better way to do this? looks kinda ugly :)

Reply:
You could use `@event.finale.winner.model_name.human.titleize` (or any participant for that matter.